server/statistics/hot_peer_cache.go

```diff
@@ -517,6 +521,7 @@ func (f *hotPeerCache) removeItem(item *HotPeerStat) {
 func coldItem(newItem, oldItem *HotPeerStat) {
 	newItem.HotDegree = oldItem.HotDegree - 1
 	newItem.AntiCount = oldItem.AntiCount - 1
+	newItem.allowAdopt = true
```

Comment:
If the item needs to be deleted, shall we allow `adopt`?

Reply:
update